asmcomp/loongarch64/arch.ml

```diff
@@ -0,0 +1,96 @@
+# 2 "asmcomp/loongarch64/arch.ml"
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                yala <zhaojunchao@loongson.cn>                          *)
+(*                                                                        *)
+(*               Copyright © 2008-2023 LOONGSON                           *)
```

Comment:
I am not sure these copyright notices are correct. I know that people at Loongson have probably been working on OCaml support since at least the Loongson 2F days, but this file (and most of the files in `asmcomp/loongarch64`, really) are extremely close to the riscv64 ones which have a completely different copyright.

I am inclined to believe that all these files are a derived work from the riscv64 compiler, and thus should at least contain its original copyright, and possibly a Loongson copyright for significant changes (if any) with the appropriate years.